### PR TITLE
Update Awake to 2.0

### PR DESCRIPTION
**Pull request by okyeron**

### CODE REVIEW

**Comment by okyeron** [comment]
Updated justmat/paths as well.

**Comment by simonvanderveldt** [comment]
(seems like your commits are done with a different e-mail than your GitHub user)

**Comment by okyeron** [comment]
Ack. Not sure how that would be, but I’ll try to sort it out

**Comment by tehn** [comment]
hello everyone.

please do not push 2.0 updates to `dust` as this repo will cease to exist after the update.

i'd also suggest people update their own scripts--- and if you're going to update someone elses' script please ask first as the update may already be in progress or finished (as is the case with my copy of `awake`)

i'll be clarifying the approach to sharing and collecting going forward--- in the meantime check out https://github.com/monome/norns/wiki/dust-2.0

in short, i suggest people begin storing scripts in their own repositories. if you're working on a shared library or engine, consider PR'ing to `norns` and shared/collab scripts can go in `we` https://github.com/monome/we

thank you for your enthusiasm for 2.0!

**Comment by okyeron** [comment]
Ok great. But we’re gonna need some starter scripts to beta test. 

What is the correct format for “our own repository” (if there is one)

**Comment by tehn** [comment]
aha, I didn't realize my fixed awake didn't make it into the pre beta
update, apologies.

proper format for a repo is described in the dust-2.0 wiki. just a folder
with lua files is fine for simple scripts.

**Comment by okyeron** [comment]
Right - but I'm a bit confused about naming things.

If I start my own new repo for norns scripts - what should that be called? It seems like this should be a consistent thing like "dust" or ? so it can be searchable/findable.  If I just call the repo "okyeron" (as it seems you're suggesting in the wiki) then I have a git path that is recursive on my own username `github.com/okyeron/okyeron/dust` ? and not very descriptive.

So yeah - I'm confused.

**Comment by tehn** [comment]
one possibility (which I intend to pursue) is the have a repo for a
particular script or series of scripts. ie mlr and meadowphysics (where my
is actually a series of variations of scripts)

I don't think having just one repo for all of your norns scripts is the
best approach to presentation. ie, not everyone is going to want to clone
your work in progress everything repo when they want to just try out one
more polished script.

re: sorting by author, that was random that it happened with dust 1.0. i
would suggest adding an author line to the about text at the top of the
list file

TODO copy/refine this text to docs/wiki (I am on a phone now)
